rskj-core/src/test/java/co/rsk/peg/PegTestUtils.java

```diff
@@ -169,6 +176,16 @@ public static Address createRandomBtcAddress() {
         return key.toAddress(RegTestParams.get());
     }
 
+    public static Address createRandomBtcAddress(NetworkParameters networkParameters) {
+        BtcECKey key = new BtcECKey();
+        return key.toAddress(networkParameters);
+    }
+
+    public static RskAddress createRandomRskAddress() {
+        ECKey key = ECKey.fromPublicOnly(new BtcECKey().getPubKey());
```

Comment:
quick comment, you don't need a btceckey to create an eckey
```suggestion
        ECKey key = new ECKey();
```

Reply:
Done